sequelize/seeders/memeolist-example-shared.js

```diff
@@ -46,11 +46,11 @@ const memeoListSchema = {
     allMemes:[Meme!]!
     comments(memeid: ID!): [Comment]!
   }
-  
+    
   type Mutation {
-    createProfile(email: String!, displayname: String!, pictureurl: String!):Profile! @hasRole(role: "realm:admin")
+    createProfile(email: String!, displayname: String!, pictureurl: String!):Profile!
     createMeme(owner: ID!, photourl: String!):Meme!
-    likeMeme(id: ID!): Boolean @hasRole(role: "realm:voter")
+    likeMeme(id: ID!): Boolean @hasRole(role: "something")
```

Comment:
Why this change?
![](https://www.mememaker.net/api/bucket?path=static/img/memes/full/2017/May/6/22/that-time-when-you-think-you-forgot-something.jpg)